docs/google-assistant-directives.rst

```diff
@@ -599,3 +599,77 @@ If you need to add output contexts to the dialog flow webhook you can use the `d
       to: "entry",
       flow: "yield",
     });
+
+
+Session Entities
+-------------------------------------------
+
+`Google Documentation <https://cloud.google.com/dialogflow/docs/entities-session>`_
+
+A session represents a conversation between a Dialogflow agent and an end-user. You can create special entities, called session entities during a session. Session entities can extend or replace custom entity types and only exist during the session that they were created for. All session data, including session entities, is stored by Dialogflow for 20 minutes.
+
+For example, if your agent has a @fruit entity type that includes "pear" and "grape", that entity type could be updated to include "apple" or "orange", depending on the information your agent collects from the end-user. The updated entity type would have the "apple" or "orange" entity entry for the rest of the session.
+
+Pre-conditions:
+In your VUI file create a LIST_OF_ENTITY tab (example: LIST_OF_FRUITS) and add the entity content to be overridden, which the one used by default.
+
+.. code-block:: javascript
+ 
+ // variables.js
+
+  export function mySessionEntity(voxaEvent: IVoxaEvent) {
+    const sessionEntityType: any = sessionEntity.fruit;
+
+    return sessionEntityType;
+  }
+  
+  // You can create a json file with the entity (mySessionEntity.json)
```

Comment:
the documentation says to create a `mySessionEntity.json` file, however looking at the tests seems like in reality this is just another key in the views file and not a separate file

Reply:
Updated please verify.